isis/src/lro/apps/lronacpho/lronacpho.h

```diff
@@ -0,0 +1,13 @@
+#ifndef lronacpho_h
+#define lronacpho_h
+
+#include "Cube.h"
+#include "lronacpho.h"
+#include "UserInterface.h"
+
+namespace Isis{
+  extern void lronacpho(Cube *iCube, UserInterface &ui, Pvl *log=nullptr);
+  extern void lronacpho(UserInterface &ui, Pvl *appLog=nullptr);
```

Comment:
kinda minor, but might as well change `appLog` here to `log` for consistency with how they're defined in the rest of the `lronacpho` signatures.

Reply:
Updated in code.